components/src/dynamo/vllm/omni/stage_worker.py

```
@@ -398,6 +453,13 @@ async def init_omni_stage(
     if config.stage_id is None:
         raise ValueError("--stage-id is required for stage worker initialization")
     stage_id: int = config.stage_id
+
+    # Register Omni's NixlConnector with vllm-omni before loading any configs
+    try:
+        register_dynamoomni_nixl_connector()
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)